plasmapy/particles/ionization_state.py

```diff
@@ -688,7 +684,7 @@ def Z_most_abundant(self) -> List[Integral]:
 
     @property
     def tol(self) -> Real:
-        """Return the absolute tolerance for comparisons."""
+        """The absolute tolerance for comparisons."""
```

Comment:
I this needs expanding.  I had to go back to the class level docstring to understand what this was.  Probably should mention it's for `numpy.isclose` and the cases it might be used for.  Or make it private.